### DISCUSSION TITLE
Rescaling .glb files

### DISCUSSION THREAD

**Discussion by GabrielArakaki** [google/model-viewer#2205] [Q&A]
### Description

Hi! I have a few thousand 3d models exported as `glb`, but they were exported by a supplier with their scale units in centimeters, so in AR we have models 100x bigger than they should.

That said, is it possible to resolve this issue in one of the following:
1 - Set the AR scale fixed, but to a % value, such as 1%?
2 - Change the scale of the AR platform to centimeters instead of meters?

Or the only option available is to change the scale in the original glb files?

Best Regards,

**Comment by jithinagiboson** [comment]
i am also having the same doubt i thought **ar-scale** property can accept scale in percentage but did not work

**Comment by elalish** [comment]
Yes, we introduced a scale attribute for this: https://modelviewer.dev/docs/#entrydocs-scenegraph-attributes-scale

**Reply by Athulkrishna-a** [reply]
Is there any documentation available for implementing scale and orientation in Flutter using this module?

**Reply by elalish** [reply]
I don't know anything about the Flutter package, but presumably they expose our standard API: https://modelviewer.dev/docs/#entrydocs-scenegraph-attributes-orientation
